src/redux/greeting.js

```diff
@@ -0,0 +1,48 @@
+// const url = 'http://127.0.0.1:3000/api/v1/messages';
+// export const fetchMessages = () => async (dispatch) => {
+//   try {
+//     const response = await fetch(url);
+//     const json = await response.data;
+//     console.log(json);
+//     dispatch({ type: 'FETCH_MESSAGES', payload: json });
+//   } catch (error) {
+//     console.error(error);
+//   }
+// };
```

Comment:
- [ ] [OPTIONAL] always keep the code base clean by removing commented codes :+1: